### DISCUSSION TITLE
https://www.yunyitang.me/Obsidian-work-note/

### DISCUSSION THREAD

**Discussion by giscus** [yunyit/yunyit.github.io#125] [Comments]
# https://www.yunyitang.me/Obsidian-work-note/

启用日期：2024-01-09  应用场景：工作笔记  自定义：荷叶绿 HEYELV #1A6840 RGB(26, 104, 64)；字体 JetBrains Mono   常用功能属性 Templater Metadata  tags

https://www.yunyitang.me/Obsidian-work-note/

**Comment by cckowin** [comment]
博主是如何记录日常项目的任务，进度，相关资料等
和每日的工作任务的，能分享一下嘛

**Reply by yunyit** [reply]
最近正好在整理换工之后的项目和笔记管理，stay tuned 我会更新的 ;)) 不过老实说，目前大部分项目和工作任务管理都在 Jira 完成了，反正公司买了Atlassian 全家桶，团队合作还挺方便
